### PR TITLE
fa translation of special built-in attributes

### PR DESCRIPTION
**Pull request by MMDShen**

### CODE REVIEW

File: src/api/built-in-special-attributes.md [hunk 1]
```diff
@@ -68,42 +70,42 @@ Denotes a [template ref](/guide/essentials/template-refs).
...
 
-  When the `is` attribute is used on a native HTML element, it will be interpreted as a [Customized built-in element](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-customized-builtin-example), which is a native web platform feature.
+  وقتی ویژگی `is` روی یک عنصر HTML بومی استفاده شود، به عنوان یک <a href="https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-customized-builtin-example">عنصر سفارشی ساخته شده</a> تفسیر میشود، که یک ویژگی بومی پلتفرم وب است.
 
-  There is, however, a use case where you may need Vue to replace a native element with a Vue component, as explained in [in-DOM Template Parsing Caveats](/guide/essentials/component-basics#in-dom-template-parsing-caveats). You can prefix the value of the `is` attribute with `vue:` so that Vue will render the element as a Vue component instead:
+  اما یک مورد استفاده وجود دارد که شما ممکن است نیاز داشته باشید که Vue یک عنصر بومی را با یک کامپوننت Vue جایگزین کند، که در احتیاطهای تجزیه تمپلیت در DOM توضیح داده شده است. شما میتوانید مقدار ویژگی `is` را با `vue:` پیشوند کنید تا Vue عنصر را به عنوان یک کامپوننت Vue رندر کند:
```
Comment (mostafa-nematpour) [inline src/api/built-in-special-attributes.md:100]:
```suggestion
  اما یک مورد استفاده وجود دارد که شما ممکن است نیاز داشته باشید که Vue یک عنصر بومی را با یک کامپوننت Vue جایگزین کند، که در [محدودیت‌های تجزیه تمپلیت در DOM](/guide/essentials/component-basics#in-dom-template-parsing-caveats) توضیح داده شده است. شما میتوانید مقدار ویژگی `is` را با `vue:‎` پیشوند کنید تا Vue عنصر را به عنوان یک کامپوننت Vue رندر کند:
```

**Review by mostafa-nematpour — CHANGES_REQUESTED** [review]
Thank you for your contribution.
Overall, it has been translated well. This review is somewhat critical, but I am sure it enhances the quality of the work.
Please check feedback.

**Comment by MMDShen** [comment]
i get it i try to fix this issues for the next article
ty for your feedback mostafa

**Review by mostafa-nematpour — COMMENTED** [review]
Thank you for your contribution.

**Pull request merged**